Tasks/RunDistributedTests/README.md

```diff
@@ -1,2 +1,103 @@
-# Run Distributed Tests Task
+# Visual Studio Test using Test Agent Task
+
+###Overview
+The Visual Studio Test using Test Agent task should be used when you want to run tests on Test Agent machine(s). Given multiple machines in the Test Machine Group, the task can do parallel distributed runs of your tests. Parallelism is at the test assembly level. To use this task, it needs to be preceded with “Visual Studio Test Agent Deployment” task.
```

Comment:
can we put more stress on the last line : "it needs to be preceeded with deploy task" .. may be underline/bold.